src/test/kotlin/domain/usecase/SuggestClothesUseCaseTest.kt

```diff
@@ -0,0 +1,120 @@
+package domain.usecase
+
+import org.damascus.data.clothes.datasource.ClothesDataSourceImp
+import org.damascus.data.clothes.repository.ClothesRepositoryImpl
+import org.damascus.data.weather.dto.CurrentWeather
+import org.damascus.domain.usecase.SuggestClothesUseCase
+
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import kotlin.test.Test
+
+class SuggestClothesUseCaseTest {
+
+    private lateinit var suggestClothesUseCase: SuggestClothesUseCase
+
+    @BeforeEach
+    fun setup() {
+        val clothesDataSource = ClothesDataSourceImp()
+        val clothesRepository = ClothesRepositoryImpl(clothesDataSource)
```

Comment:
Should use a `mockk()` to protect the integrity of the test and prevent it from being flaky.